tests/e2e/README.md

```
@@ -140,13 +148,17 @@ PWDEBUG=1 pytest tests/e2e/ -v
 
 ## Output
 
-Abbreviated pass shape (41 tests across the five files):
+Abbreviated pass shape (47 tests across the five files):
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)